Python/bytecodes.c

```diff
@@ -4199,6 +4207,20 @@ dummy_func(
             frame->instr_ptr = (_Py_CODEUNIT *)instr_ptr;
         }
 
+        tier2 op(_DEOPT, (--)) {
+            EXIT_TO_TIER1();
+        }
+
+        tier2 op(_SIDE_EXIT, (--)) {
+            EXIT_TO_TRACE();
+        }
+
+        tier2 op(_ERROR_POP_N, (values[oparg] --)) {
+            (void)values;
```

Comment:
Would `unused` work here?

```suggestion
        tier2 op(_ERROR_POP_N, (unused[oparg] --)) {
```